bin/cml/comment/create.js

```diff
@@ -16,6 +16,12 @@ exports.options = kebabcaseKeys({
     description:
       'Post to an existing PR/MR associated with the specified commit'
   },
+  issue: {
+    type: 'string',
```

Comment:
Should it take a number instead?

```suggestion
    type: 'number',
```